missing hdf5::hdf5_hl in node tests

For older version of libhdf5 (1.10.0) one needs to add  hdf5::hdf5_hl to the link target of node tests. Otherwise we get
```
/usr/bin/x86_64-linux-gnu-ld: CMakeFiles/node_test.dir/dataset_direct_chunk_test.cpp.o: undefined reference to symbol 'H5DOwrite_chunk@@HDF5_SERIAL_1.8.11'
//usr/lib/x86_64-linux-gnu/libhdf5_serial_hl.so.100: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

--- a/test/node/CMakeLists.txt
+++ b/test/node/CMakeLists.txt
@@ -35,6 +35,7 @@ target_link_libraries(
    PRIVATE
        h5cpp
        hdf5::hdf5
+       hdf5::hdf5_hl
        example_lib
 )
 target_compile_definitions(node_test PRIVATE CATCH_CONFIG_ENABLE_BENCHMARKING)

